weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/AddToDatasetDrawer.tsx

```diff
@@ -0,0 +1,309 @@
+import {Box, Typography} from '@mui/material';
```

Comment:
Idk if this is just my machine, but the trace table interactions feel slow in this branch